src/main/java/core/basesyntax/writer/FileWriterImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.writer;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String resultingReport, String filePath) {
```

Comment:
According to the checklist, avoid hardcoding file paths. It's better to receive the file path as a parameter to this method.